src/main/java/com/sonyericsson/jenkins/plugins/bfa/model/FailureCause.java

```diff
@@ -663,6 +616,61 @@ public String getDisplayName() {
             return "";
         }
 
+        /**
+         * Form validation for {@link #description}. Checks for not empty and not "Description..."
+         *
+         * @param value the form value.
+         * @return {@link hudson.util.FormValidation#ok()} if everything is well.
+         */
+        @POST
```

Comment:
I think `@RequirePOST` would be better suited here, so the caller gets a real error instead of a not found.